unicorecmsgem/static/css/style.css

```diff
@@ -133,6 +133,16 @@ white-space: pre\9; /* IE7+ */
 .declaration .listing p { margin: 0; padding: 2px 0 0 0; color: #888888; }
 .declaration .pagination a { font-weight: bold; color: #39c0c0; }
 .declaration .detail { padding: 10px; }
+/*Search Box*/
```

Comment:
Doesn't this already exist somewhere as `.form-search`? Add a `margin-top: 1em;` to that.